components/Map/Map.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+
+import MapView, { Marker } from 'react-native-maps';
+import { StyleSheet, Text, View, Button, TextInput } from 'react-native';
+
+export default function Map() {
+
+let events = [
+    {
+      id: 1,
+      date: 20200407,
+      time: 1700,
+      unix_time: 1586278800,
+      activity: 'Volleyball',
+      location: 'Wash Park',
```

Comment:
We're going to need to find a way to tie names of parks with their specific locations for the map to recognize. Not sure if this happens automatically or if it's something we need to add code to do.

Reply:
Yeah we will have to discuss that with BE. I believe there's functionality they can add to find the coordinates for places, but i'm not 100% certain, so we can reach out to them about that